src/main/scala/com/fulcrumgenomics/bam/ClipBam.scala

```diff
@@ -188,9 +188,7 @@ class ClipBam
 
     val (numExtendingPastMateStartReadOne, numExtendingPastMateStartReadTwo) = {
       if (clipBasesPastMate && r1.isFrPair) {
-        val clip1 = this.clipper.clipExtendingPastMateEnd(rec=r1, mateEnd=r2.end)
-        val clip2 = this.clipper.clipExtendingPastMateEnd(rec=r2, mateEnd=r1.end)
-        (clip1, clip2)
+        this.clipper.clipExtendingPastMateEnds(rec=r1, mate=r2)
```

Comment:
I think this new change is OK and the docs **_do not_** need updating:


```console
--clip-bases-past-mate
Clip reads in FR pairs that sequence past the far end of their mate.
```

Since "far end" is unspecified with regard to the mate's soft-clipping or not.